sources/packages/api/src/route-controllers/institution-locations/models/institution-location.dto.ts

```diff
@@ -64,3 +65,14 @@ export interface InstitutionLocationsSummaryDto {
     postalCode: string;
   };
 }
+
+/**
+ * Possible status for a designation location agreement.
+ */
+export enum DesignationLocationAgreementStatus {
+  Designated = "Designated",
+  /**
+   * The designation agreement status is not designated
```

Comment:
since you have added a comment to `NotDesignated `  maybe you can add for `Designated ` too